### ISSUE TITLE
Unsupported PG Syntax during the Migration from any source database

### ISSUE THREAD

**Issue by priyanshi-yb** [yugabyte/yb-voyager#1625]
This ticket serves as a placeholder for tracking issues identified by the analyze-schema command, specifically, those categorized as `Unsupported PG syntax`. These issues pertain to statements in the dumped schema that are incompatible with PostgreSQL syntax and will result in errors during the import schema process when targeting a YugabyteDB database.

**Comment by BrettHoyer** [comment]
This has been flagged for the statement below:
```
CREATE INDEX index_import_failures_on_external_identifiers ON public.import_failures USING btree (external_identifiers) WHERE (external_identifiers <> '{}'::jsonb);
```

**Comment by hbhanawat** [comment]
@priyanshi-yb do we need this issue?

**Comment by priyanshi-yb** [comment]
> @priyanshi-yb do we need this issue?

Yes we are this GitHub issue for putting it in the analyze Issue as a blanket ticket for these cases.
